RecoParticleFlow/PFProducer/plugins/PFProducer.cc

```diff
@@ -317,6 +328,10 @@ void PFProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions)
   desc.add<edm::InputTag>("muons", edm::InputTag("muons1stStep"));
   desc.add<bool>("postMuonCleaning", true);
 
+  // vetoEndcap flag and pf candidates for vetoes
+  desc.add<bool>("vetoEndcap", false);
+  desc.add<edm::InputTag>("vetoes", edm::InputTag("pfTICL"));
```

Comment:
use ifValue here https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#IfExists
```suggestion
  desc.ifValue(edm::ParameterDescription<bool>("vetoEndcap", false, true),
               true >> edm::ParameterDescription<edm::InputTag>("vetoes", {"pfTICL"}, true));
```
I did not check if this compiles

Reply:
adding also `false >>` it compiles. thx.